logo/license

```diff
@@ -0,0 +1,65 @@
+Apache License Version 2.0, January 2004 http://www.apache.org/licenses/
```

Comment:
I don't think we need this file since we already have a LICENSE file that covers the entire project in the root path.

Reply:
I will remove it then